docker-compose.yml

```diff
@@ -1,10 +1,24 @@
 version: "3.9"
 
-services: 
+services:
   rabbitmq:
     image: rabbitmq:3.10-management
     ports:
       - 5672:5672
       - 15672:15672
     volumes:
       - ./rabbitmq_config:/etc/rabbitmq
+
```

Comment:
Couldn't get this to work on an M1 mac, we can go through it though.

Reply:
You may need to update and set some settings https://github.com/microsoft/mssql-docker/issues/668#issuecomment-1436802153